flax/experimental/nnx/tests/nn/test_attention.py

```diff
@@ -0,0 +1,10 @@
+import jax.numpy as jnp
+
+from flax.experimental import nnx
+
+
+class TestMultiHeadAttention:
+  def test_basic(self):
+    module = nnx.MultiHeadAttention(2, 3, 6, rngs=nnx.Rngs(0))
+    y = module(jnp.ones((1, 7, 3)))
+    assert y.shape == (1, 7, 6)
```

Comment:
Can we port over tests from [`tests/linen_attention_test.py`](https://github.com/google/flax/blob/main/tests/linen/linen_attention_test.py)